stellargraph/core/graph.py

```diff
@@ -134,14 +115,21 @@ class StellarGraph(metaclass=StellarGraphFactory):
 
     """
 
+    def __init__(self, *args, **kwargs):
+        # Avoid a circular import
+        from .graph_networkx import NetworkXStellarGraph
+
+        is_directed = kwargs.pop("is_directed", False)
```

Comment:
Do we need to handle this particular parameter explicitly like this? Seems like `NetworkXStellarGraph` should default to `is_directed=False` anyway?